runtime/pack/dist/opt/netrw/plugin/netrwPlugin.vim

```diff
@@ -15,7 +15,7 @@ if &cp || exists("g:loaded_netrwPlugin")
     finish
 endif
 
-let g:loaded_netrwPlugin = "v175"
+let g:loaded_netrwPlugin = "v181"
```

Comment:
This should be changed to `v184` in the commit that changed the version in netrw.vim to `v184`.